test/integration/osl-gitlfs/inspec/osl_gitlfs_spec.rb

```diff
@@ -1,3 +1,6 @@
+pseudoGroup = 'nobody'
+pseudoGroup = 'nogroup' if os.family == 'debian'
```

Comment:
Do the same as I suggested above please.

Reply:
Pushed changes